AUTHORS.txt

```diff
@@ -49,4 +49,5 @@ Jack Van Sambeek
 JC Arbelbide
 Florian Jünger
 Benedikt Moneke
+Brandon Altieri
```

Comment:
Until someone gets around and fixes this duplication, please also add your name to docs/about/authors.rst